src/aiodynamo/http/types.py

```diff
@@ -23,6 +23,4 @@ class RequestFailed(Exception):
     inner: Exception
 
 
-class HttpImplementation(Protocol):
-    async def __call__(self, request: Request) -> Response:
-        ...
+HttpImplementation = Callable[[Request], Awaitable[Response]]
```

Comment:
`Protocol` import can be removed

Reply:
and maybe if all the uses of Protocol are done, it can be removed from `_compat`